packages/web-components/src/global/main.css

```diff
@@ -1,4 +1,5 @@
 @import 'variables.css';
+@import '../components/va-breadcrumbs/va-breadcrumbs-slot.css';
```

Comment:
I like this more than what I did with just putting the `va-process-list` selector directly in here, but we may want to think about what we can do to limit this kind of thing from becoming a burden to maintain.